### ISSUE TITLE
Can not reproduce the results on hERG dataset. 

### ISSUE THREAD

**Issue by sjchasel** [Niuchx/HimNet#2]
Hi, 
I run the train_herg.py but only achieve 0.3 AUC, which is far from the reported result in the paper.
![截屏2023-08-28 上午11 14 34](https://github.com/Niuchx/HimNet/assets/57445585/ee9fc516-f49b-408f-baa8-5906ee589dd8)

**Comment by Niuchx** [comment]
Hi, 
Sorry about that. In general, you can try different hyperparameter settings on your machine which include the number of graph/node memory blocks, learning rate and so on. I have updated my hyperparameters in the "train_herg.py".
